src/d3a/models/strategy/external_strategies/load.py

```diff
@@ -127,15 +134,17 @@ def _bid(self, payload):
             return
         try:
             arguments = json.loads(payload["data"])
-            assert set(arguments.keys()) == {'price', 'energy', 'transaction_id'}
+            assert all(arg in {'price', 'energy', 'replace_existing'} for arg in arguments.keys())
+
             arguments['buyer_origin'] = self.device.name
         except Exception:
             self.redis.publish_json(
                 bid_response_channel,
                 {"command": "bid",
-                 "error": "Incorrect bid request. Available parameters: (price, energy).",
-                 "transaction_id": transaction_id}
-            )
+                 "error": (
+                     "Incorrect offer request. "
+                     f"Available parameters: {'price', 'energy', 'replace_existing'}."),
+                 "transaction_id": transaction_id})
```

Comment:
Why this change? It is indeed a bid request, not an offer request.

Reply:
👍 Mistake on my side, probably while replicating the logic